db/migrate/20170712033920_add_language_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLanguageToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :language, :string
```

Comment:
Out of interest: why store this value rather than detecting it on the server-side each time?